### ISSUE TITLE
2.28.	Outstanding:MeD: Focus not visible (2.4.7) 

### ISSUE THREAD

**Issue by falcochip** [MeD-DMC/Snap-framework-MeD#44]
When tabbing through these pages it often skips a radio button if it involves a choice of two. You can focus on one radio button, but tabbing again will bring you to the Submit button.
I also ran jaws on a few of these pages and it does not read out the second radio buttons options. It clearly said “radio button 1 of 1” which isn’t true. There are two choices.
